src/components/Create/components/DurationInput/DurationInput.tsx

```diff
@@ -0,0 +1,70 @@
+import { Trans } from '@lingui/macro'
+import { Input, Select } from 'antd'
+import { DurationUnitsOption } from 'constants/time'
+import { ChangeEvent, CSSProperties } from 'react'
+
+export interface DurationInputValue {
+  duration: number | undefined
+  unit: DurationUnitsOption
+}
+
+export const DurationInput: React.FC<{
+  style?: CSSProperties
+  value?: DurationInputValue
+  onChange?: (value: DurationInputValue | undefined) => void
+}> = ({ style, value, onChange }) => {
+  const handleChange = (incoming: DurationInputValue) => {
+    onChange?.(incoming)
+  }
+
+  const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const unit = value?.unit ?? 'days'
+    if (!e.target.value) {
+      handleChange({ duration: undefined, unit })
+      return
+    }
+
+    const duration = parseInt(e.target.value)
+    if (isNaN(duration)) return
+    if (duration <= 0) {
+      handleChange({ duration: 1, unit })
+      return
+    }
+
+    handleChange({ duration: duration, unit })
+  }
+
+  return (
+    <div style={{ display: 'flex', gap: '0.625rem', ...style }}>
+      <Input
+        placeholder="14"
```

Comment:
**question** I think we want `14` as the default value rather than placeholder.

Reply:
I reckon lets defer this one as its non-trivial unfortunately. State data with the redux stuff is PIA